source/RuuviTagApp.mc

```diff
@@ -27,6 +28,39 @@ class RuuviTagApp extends Application.AppBase {
     bleDelegate_ = new RuuviTagBleDelegate(method(:onRuuviTagData));
   }
 
+  private function verifySensorProperty (sensorMacAddress, sensorAlias) {
+    var sensorMacAddressCA = sensorMacAddress.toCharArray();
+
+    // verify if format of user-configurable hardware address is reasonable
+    if ((sensorMacAddress.length() == 14) && 
+        (sensorMacAddressCA[2] ==
+         sensorMacAddressCA[5] ==
+         sensorMacAddressCA[8] ==
+         sensorMacAddressCA[11] == ':')) {
+        return true;
+      }
+    return false;
+  }
+
+  function onStart(state) {
+    for (var i = 1; i < 6; i++) {

```

Comment:
As a matter of convention in 0-based-index languages like Monkey C, could we express this as "var i = 0; i < 5; i++"? There's generally less confusion in interpreting this number of iterations this way. If you wish to leave the sensor indices as 1-based, just create a "var sensorIndex = i + 1;" below and use that in place of i.